server/upgrade_risks_prediction.go

```diff
@@ -125,13 +130,64 @@ func (server *HTTPServer) upgradeRisksPrediction(writer http.ResponseWriter, req
 	}
 }
 
+// upgradeRisksPredictionMultiClusters sends a response with a list of predictions for each cluster.
+// Each prediction will have a result (true or false) and a list of the alerts and operator conditions
+// that were taken into account for non-recommended upgrades.
+func (server *HTTPServer) upgradeRisksPredictionMultiCluster(writer http.ResponseWriter, request *http.Request) {
+	if request.ContentLength <= 0 {
+		handleServerError(writer, &NoBodyError{})
+		return
+	}
+
+	var clusterList ctypes.ClusterListInRequest
+
+	// try to read cluster list from request parameter
+	err := json.NewDecoder(request.Body).Decode(&clusterList)
+	if err != nil {
+		handleServerError(writer, err)
+		return
+	}
+
+	if len(clusterList.Clusters) > MaxAllowedClusters {
+		handleServerError(writer, &BadBodyContent{})
```

Comment:
would it be worth adding a more descriptive error message so that it's obvious it's because of too many clusters?

Reply:
@Bee-lee PTAL: https://github.com/RedHatInsights/insights-results-smart-proxy/pull/1210/commits/7b088f740c5437f17465c345daeaf7940e695274